Source/Core/Models/CardScheme.swift

```diff
@@ -8,7 +8,6 @@
 import Checkout
 
 public enum CardScheme {
```

Comment:
You may have to update the Unit tests. 
<img width="1503" alt="Screenshot 2022-08-16 at 15 32 37" src="https://user-images.githubusercontent.com/59569588/184906220-a791bbfc-ac2b-4bda-8ba7-e318acb99260.png">